src/gen-api-models.ts

```diff
@@ -3,21 +3,15 @@
 import * as fs from "fs-extra";
 import { ITuple2, Tuple2 } from "italia-ts-commons/lib/tuples";
 import * as nunjucks from "nunjucks";
+import { OpenAPIV2, OpenAPI } from "openapi-types";
```

Comment:
```suggestion
import { OpenAPI, OpenAPIV2 } from "openapi-types";
```
named imports must be alphabetized